offchainservice/src/main/java/pl/edu/agh/blockchain/offchainservice/service/EMailServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package pl.edu.agh.blockchain.offchainservice.service;
+
+import com.sendgrid.*;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+
+@Service
+public class EMailServiceImpl implements EMailService {
+
+    @Override
+    public void sendEmail(String email, String token) throws IOException {
+        Email from = new Email("blockchain@agh.edu.pl");
+        String subject = "Sending with SendGrid is Fun";
```

Comment:
Let's set the subject to be something like: 'Voting Dapp verification token from AGH Blockchain'